questions/node-error-first-callback.md

```diff
@@ -1,4 +1,4 @@
-### NodeJS uses a callback pattern in many instances where if an error were returned it will pass it as the first argument to the callback. What are the advantages of this pattern?
+### NodeJS often uses a callback pattern if an error is encountered during execution. This error is passed as the first argument to the callback. What are the advantages of this pattern?
```

Comment:
you should have left 'where' after 'pattern' before if because now you have a transition with no condition which is confusing:

> callback pattern if an error

versus
> callback pattern where if an error

Reply:
Unnecessary punctuation since there is no transition of context

>during execution. This error

versus
>during execution this error

Reply:
@skacat31. Thanks for the review, I'll modify your second point, and add a comma instead. On the former I find _where_ a little unusual in this sentence, that could just be American vs. British English, I'm happy to defer to you.

Reply:
@milesburton To be fair, that where could easily also be an and